unittests/resources/checks/paramcheck.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2016-2020 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I wouldn't use a separate file for the tests but rather define them directly in the unit test file.

Reply:
The tests can be really minimal and since we don't plan to run them, they can go in the unit test file directly.

Reply:
Makes sense. Moved theses classes to `unittests/test_parameters.py`